FileContentOrdering.md

```diff
@@ -62,3 +62,47 @@ internal func updateUnalliedHouses(_ houses: [House]) {
 private maximumHouseSize: Int = 50
 ```
 
+## Convention - Constants
+
+Use nested structs to group constants when possible.
```

Comment:
A few questions here:
- We explicitly call out nested structs, but our example doesn't seem to use nested structs. Maybe just say we should use structs instead of saying nested structs?
- Should we be more specific and say the grouping should be logical, not just based on the structure of the constants?
    - I'd like to avoid `struct Constants {}` or `struct Strings {}` types of groupings that aren't actual logical groupings and more just dumping grounds for things.

Reply:
I think the original intention of the "nested struct" is to demonstrate that the struct is a nested type within the class, but I can reword it to "Create nested structs within classes to group constants when possible; the grouping should be based on logical use instead of structure or type of the constants".